pinot-segment-local/src/main/java/org/apache/pinot/segment/local/startree/v2/builder/MultipleTreesBuilder.java

```diff
@@ -90,6 +90,13 @@ public MultipleTreesBuilder(List<StarTreeV2BuilderConfig> builderConfigs, File i
     _metadataProperties =
         CommonsConfigurationUtils.fromFile(new File(_segmentDirectory, V1Constants.MetadataKeys.METADATA_FILE_NAME));
     _separator = getSeparator();
+    // log the updated star-tree configs
+    StringBuilder logUpdatedStarTrees = new StringBuilder();
+    logUpdatedStarTrees.append("Updated star-tree configs :");
+    for (StarTreeV2BuilderConfig startree : _builderConfigs) {
+      logUpdatedStarTrees.append("\n").append(startree);
+    }
+    LOGGER.debug(logUpdatedStarTrees.toString());
```

Comment:
Check whether debug log is enabled, then collect these info. Currently we always collect info even when debug log is disabled